src/_tests/libs/runtime-utils.test.js

```diff
@@ -0,0 +1,189 @@
+import { getCurrentApp, getCurrentAppIncludingDeleting, getCurrentPersistentDisk, getDiskAppType } from 'src/libs/runtime-utils'
+
+
+const cromwellRunning = {
+  appName: 'terra-app-83f46705-524c-4fc8-xcyc-97fdvcfby14f',
+  appType: 'CROMWELL',
+  auditInfo: {
+    creator: 'cahrens@gmail.com', createdDate: '2021-11-28T20:28:01.998494Z', destroyedDate: null, dateAccessed: '2021-11-28T20:28:01.998494Z'
+  },
+  diskName: 'saturn-pd-693a9707-634d-4134-bb3a-xyz73cd5a8ce',
+  errors: [],
+  googleProject: 'terra-test-e4000484',
+  kubernetesRuntimeConfig: { numNodes: 1, machineType: 'n1-highmem-8', autoscalingEnabled: false },
+  labels: {},
+  proxyUrls: { 'cromwell-service': 'https://leonardo-fiab.dsde-dev.broadinstitute.org/fd0cfbb14f/cromwell-service/swagger/cromwell.yaml' },
+  status: 'RUNNING'
+}
+
+// Newer than cromwellRunning.
+const cromwellProvisioning = {
+  appName: 'terra-app-73f46705-524c-4fc8-ac8c-07fd0cfbb14f',
+  appType: 'CROMWELL',
+  auditInfo: {
+    creator: 'cahrens@gmail.com', createdDate: '2021-11-29T20:28:01.998494Z', destroyedDate: null, dateAccessed: '2021-11-29T20:28:01.998494Z'
+  },
+  diskName: 'saturn-pd-693a9707-634d-4134-bb3a-cbb73cd5a8ce',
+  errors: [],
+  googleProject: 'terra-test-e4000484',
+  kubernetesRuntimeConfig: { numNodes: 1, machineType: 'n1-highmem-8', autoscalingEnabled: false },
+  labels: {},
+  proxyUrls: { 'cromwell-service': 'https://leonardo-fiab.dsde-dev.broadinstitute.org/fd0cfbb14f/cromwell-service/swagger/cromwell.yaml' },
+  status: 'PROVISIONING'
+}
+
+const galaxyRunning = {
+  appName: 'terra-app-69200c2f-89c3-47db-874c-b770d8de737f',
+  appType: 'GALAXY',
+  auditInfo: {
+    creator: 'cahrens@gmail.com', createdDate: '2021-11-29T20:19:13.162484Z', destroyedDate: null, dateAccessed: '2021-11-29T20:19:13.162484Z'
+  },
+  diskName: 'saturn-pd-026594ac-d829-423d-a8df-76fe96f5b4e7',
+  errors: [],
+  googleProject: 'terra-test-e4000484',
+  kubernetesRuntimeConfig: { numNodes: 1, machineType: 'n1-highmem-8', autoscalingEnabled: false },
+  labels: {},
+  proxyUrls: { galaxy: 'https://leonardo-fiab.dsde-dev.broadinstitute.org/a-app-69200c2f-89c3-47db-874c-b770d8de737f/galaxy' },
+  status: 'RUNNING'
+}
+
+// Newer than galaxyRunning
+const galaxyDeleting = {
+  appName: 'terra-app-71200c2f-89c3-47db-874c-b770d8de22g',
+  appType: 'GALAXY',
+  auditInfo: {
+    creator: 'cahrens@gmail.com', createdDate: '2021-11-30T20:19:13.162484Z', destroyedDate: null, dateAccessed: '2021-11-30T20:19:13.162484Z'
+  },
+  diskName: 'saturn-pd-1236594ac-d829-423d-a8df-76fe96f5897',
+  errors: [],
+  googleProject: 'terra-test-e4000484',
+  kubernetesRuntimeConfig: { numNodes: 1, machineType: 'n1-highmem-8', autoscalingEnabled: false },
+  labels: {},
+  proxyUrls: { galaxy: 'https://leonardo-fiab.dsde-dev.broadinstitute.org/a-app-69200c2f-89c3-47db-874c-b770d8de737f/galaxy' },
+  status: 'DELETING'
+}
+
+const mockApps = [cromwellProvisioning, cromwellRunning, galaxyRunning, galaxyDeleting]
+
```

Comment:
We tend to have a naming scheme like below

galaxyDiskReady
galaxyDiskDeleting
cromwellDiskUnattached

for more consistency. What do you think?

Reply:
These are the apps, not the disks. The disks have their own statuses which are already in their names.

Reply:
Oh sorry, I was referring to the disk objects below.